src/NWCClient.ts

```diff
@@ -47,6 +48,16 @@ export type Nip47GetInfoResponse = {
   notifications?: Nip47NotificationType[];
 };
 
+export type Nip47GetBudgetResponse =
+  | {
+      used_budget: number; // msats
+      total_budget: number; // msats
+      renews_at?: number; // timestamp
```

Comment:
This is defined as `number | undefined` but if `renewal_period` is never we get `renews_at: null` 

![Screenshot 2024-10-22 at 1 20 02 PM](https://github.com/user-attachments/assets/615fed2e-cf97-4987-81ee-adb9e0c6eb71)

Reply:
Good catch, I think we should fix this in Alby Hub to omit `renews_at` in the json.

Reply:
Fixed here: https://github.com/getAlby/hub/pull/726/commits/634e54d778bee73c0c4f73dceffa7d05b7a1d21c

Reply:
Resolved by https://github.com/getAlby/hub/pull/726/commits/634e54d778bee73c0c4f73dceffa7d05b7a1d21c